### PR TITLE
fix: disconnect wallet

### PR DESCRIPTION
**Pull request by azmimuhammad**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-2678

### Changelog/Description 🗒️
- fix can't disconnect wallet

### WARNING ⚠️
-

### CODE REVIEW

File: src/components/HeaderWallet.vue [hunk 1]
```diff
@@ -118,6 +120,10 @@ export default {
       metamaskWalletAddress: (state) => state.metamask.metamaskWalletAddress,
       metamaskWalletBalance: (state) => state.metamask.metamaskWalletBalance,
       lastEventData: (state) => state.substrate.lastEventData
+    }),
+
+    ...mapGetters({
+      pair: "substrate/wallet"
```
Comment (ciniiia) [inline src/components/HeaderWallet.vue:126]: what is this for ? this already computed above as this.wallet

File: src/components/HeaderWallet.vue [hunk 2]
```diff
@@ -193,9 +200,17 @@ export default {
       }
     },
 
-    disconnectWallet() {
-      this.metamaskAddress = ""
-      this.clearWallet()
+    async disconnectWallet() {
+      this.metamaskLoading = true
+      try {
+        await setEthAddress(this.api, this.pair, "")
```
Comment (ciniiia) [inline src/components/HeaderWallet.vue:206]: this won't work as in polkadot you cannot input empty string for the `ethAddress`. 
As discuss with @hilyds , for disconnect wallet, you just need to clear the local storage.

**Review by ciniiia — CHANGES_REQUESTED** [review]

File: src/components/HeaderWallet.vue (hunk 1 above)
Comment (azmimuhammad) [reply src/components/HeaderWallet.vue]: updated using this.wallet

File: src/components/HeaderWallet.vue (hunk 2 above)
Comment (azmimuhammad) [reply src/components/HeaderWallet.vue]: I made changes according to what has been discussed, save the metamask address in local storage

**Review by ciniiia — APPROVED** [review]

**Pull request merged**